kafka/client.go

```diff
@@ -0,0 +1,13 @@
+package kafka
+
+import (
+	messaging "github.com/veritone/go-messaging-lib"
+)
+
+// This interface is for kafka usage specifically. This is to give further details on kafka consumer's specific implementations
+
+// ClientConsumer defines interface for kafka consumer
+type ClientConsumer interface {
+	messaging.Consumer
+	MarkOffset(messaging.Event, string) error
+}
```

Comment:
why not just call this interface `Consumer`?

Reply:
It conflicts with the method named `Consumer` in https://github.com/veritone/go-messaging-lib/blob/master/kafka/consumer.go#L60

Reply:
I could just change the name of that method while I'm at it.